cmd/mayactl/app/command/volume_test.go

```diff
@@ -49,32 +55,38 @@ func TestGetCASType(t *testing.T) {
 func TestGetClusterIP(t *testing.T) {
 	tests := map[string]struct {
 		expectedOutput string
-		Volume         CASVolume
+		Volume         VolumeInfo
 	}{
 		"Fetching ClusterIP from openebs.io/cluster-ips": {
-			Volume: CASVolume{
-				ObjectMeta: metav1.ObjectMeta{
-					Annotations: map[string]string{
-						"openebs.io/cluster-ips": "192.168.100.1",
+			Volume: VolumeInfo{
+				Volume: v1alpha1.CASVolume{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: map[string]string{
+							"openebs.io/cluster-ips": "192.168.100.1",
+						},
 					},
 				},
 			},
 			expectedOutput: "192.168.100.1",
 		},
 		"Fetching ClusterIP from vsm.openebs.io/cluster-ips": {
-			Volume: CASVolume{
-				ObjectMeta: metav1.ObjectMeta{
-					Annotations: map[string]string{
-						"vsm.openebs.io/cluster-ips": "192.168.100.1",
+			Volume: VolumeInfo{
+				Volume: v1alpha1.CASVolume{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: map[string]string{
+							"vsm.openebs.io/cluster-ips": "192.168.100.1",
+						},
```

Comment:
Add both `"openebs.io/cluster-ips": "192.168.100.1"` and `"vsm.openebs.io/cluster-ips": "192.168.100.1"` in VolumeInfo object so test which get s selected.

Reply:
sure

Reply:
Thanks

Reply:
The functions getClusterIP amd getControllerStatus does not try to fetch from specs but only
from anmotations.
Is it so that for every case these values will always be present in annotations?

Reply:
No it will be present on annotation everytime